### ISSUE TITLE
client.bin always missing after build

### ISSUE THREAD

**Issue by agent47007** [quasar/QuasarRAT#727]
hello,
i added the folder as an exception to comodo antivirus 10. 
but i noticed that anytime i build a solution when my anti-virus is on i get a successful build but without the client.bin file
but when i build a solution when i have disabled of my antivirus setting i get the following error below.
i have already tried and tried package restore so many times and play around with package manager for hours and days without any resolve.
i also disabled most of my windows defender settings. can you do a build and send me
thanks
Severity	Code	Description	Project	File	Line	Suppression State
Error		The command ""C:\Users\agent47\achike\QuasarRAT\packages\ILMerge.2.14.1208\tools\ILMerge.exe" /out:"C:\Users\agent47\achike\QuasarRAT\bin\Release\Client.all.exe" "C:\Users\agent47\achike\QuasarRAT\bin\Release\Client.exe" "C:\Users\agent47\achike\QuasarRAT\bin\Release\protobuf-net.dll" "C:\Users\agent47\achike\QuasarRAT\bin\Release\Gma.System.MouseKeyHook.dll" "C:\Users\agent47\achike\QuasarRAT\bin\Release\Quasar.Common.dll" /target:WinExe /wildcards /internalize
copy "C:\Users\agent47\achike\QuasarRAT\bin\Release\Client.all.exe" "C:\Users\agent47\achike\QuasarRAT\bin\Release\client.bin" /Y" exited with code 1.	Quasar.Client	
Severity	Code	Description	Project	File	Line	Suppression State
Warning	CS0169	The field 'Client._typeIndex' is never used	Quasar.Client	C:\Users\agent47\achike\QuasarRAT\Quasar.Client\Networking\Client.cs	197	Active

**Comment by mystyq** [comment]
Disable Windows Defender Real-time protection

**Comment by 15w34r0n3D4y** [comment]
Solution is:

First download/manage nuget package for Client called ILMerge.Tools. (installed in QuasarRAT/packages)
After it click Properties on Client then Build Events and  edit Post-build. (from \bin\Release\Client.all.exe to \bin\Release\Client.exe). 

That's it.

@agent47007

**Comment by 15w34r0n3D4y** [comment]
> 
> 
> That's not required if you would just disable the AV.

Yes I disable it, but when I downloading file from GH and FF detect it like virus, I got error as user @agent47007 and fix like it, but client doesn't work. Then I use other method and clone from GH and try again, now everything works (client and server) without errors. I see client.exe and client.all.exe got diff size file. 

Thank you 👍

**Comment by MaxXor** [comment]
Yes they have different file sizes because Client.all includes all required dependencies (dlls).
